src/main/grammars/org/nest/Commons.mc4

```diff
@@ -54,7 +54,7 @@ grammar Commons extends de.monticore.types.Types, org.nest.Units {
          | FunctionCall
          | BooleanLiteral // true & false;
          | NumericLiteral Variable
-	 | NumericLiteral
+	       | NumericLiteral
```

Comment:
I have no idea what is going wrong here, I fixed the spacing again (and it looked okay on my machine both times).
I removed all whitespaces in the area and re-entered normal spaces. Still does not seem to help it.